### PR TITLE
misc: Updates some documentation and refactors a little bit

### PR DESCRIPTION
**Pull request by CristhianMotoche**
# Changes
- doc: Update cabal Intro
- ref: Move helpers to new module

### CODE REVIEW

**Review by FranzGB — APPROVED** [review]
LGTM!

File: dotenv.cabal [hunk 1]
```diff
@@ -5,38 +5,37 @@ homepage:            https://github.com/stackbuilders/dotenv-hs
...
   .
-  See the <https://github.com/stackbuilders/dotenv-hs Github>
+  See the <https://github.com/stackbuilders/dotenv-hs#readme Github>
   page for more information on this package.
 license:             MIT
 license-file:        LICENSE
 author:              Justin Leitgeb
 maintainer:          hackage@stackbuilders.com
-copyright:           2015-2020 Stack Builders Inc.
+copyright:           2015-Present Stack Builders Inc.
```
Comment (sestrella) [inline dotenv.cabal:38]: 👏🏼

File: src/Configuration/Dotenv/Types.hs [hunk 2]
```diff
@@ -39,26 +35,3 @@ defaultConfig =
     , configVerbose = False
     , allowDuplicates = True
     }
-
-
-newtype ReaderT r m a = ReaderT { runReaderT :: r -> m a }
```
Comment (sestrella) [inline src/Configuration/Dotenv/Types.hs:44]: @CristhianMotoche nice clean-up!

**Review by sestrella — APPROVED** [review]
@CristhianMotoche LGTM

**Pull request merged**